dt/cloudops/openstack-telemetry_DT.md

```diff
@@ -0,0 +1,54 @@
+# RHOSO Deployed Topology DFG-cloudops-metmon-openstack-telemetry-18.0-3cont_2comp_final-ipv4-rhosfips
```

Comment:
the naming suggests 3 controllers, 2 compute nodes and enabling fips (which *should* be fine).

Reply:
about the naming, please check the changes to the default template.  I would also use a less, uhm, detailed, name (please check the other proposals).

Reply:
let's use something like observbility for the name and not use a name that would imply a certain configuration (like number of controllers, or fips enabled/disabled).

Reply:
Done

Reply:
@mrunge @tosky I have committed the changes